packages/lab/stories/badge.stories.tsx

```diff
@@ -62,3 +65,14 @@ WordsBadge.args = {
   badgeContent: 1,
   children: "Lorem Ipsum",
 };
+
+export const CustomStyling: ComponentStory<typeof Badge> = () => (
+  <div style={{ display: 'flex', gap: 5 }}>
+    <ToolkitProvider theme={["success", 'newapp']}>
```

Comment:
we need to keep "light" or "dark" in the theme to get the rest of the styles (in storybook, we luckily have them coming from higher up)
then, let's add 2 Badge's to both light and dark, one with className 'success' and one with 'error'! :)